server/controllers/account.coffee

```diff
@@ -122,7 +122,7 @@ module.exports =
             else    
                 user = users[0]
                 res.status(200).send 
-                    token: base32.encode(user.encryptedOtpKey).toString()
+                    token: base32.encode(new Buffer(user.encryptedOtpKey, 'hex')).toString()
```

Comment:
Could you please make this line less than 80-char long?

For example, by replacing

```
                user = users[0]
                    res.status(200).send 
                        token: base32.encode(new Buffer(user.encryptedOtpKey, 'hex')).toString()
```

with

```
                user = users[0]
                buffer = new Buffer(user.encryptedOtpKey, 'hex')
                res.status(200).send 
                    token: base32.encode(buffer).toString()